app.py

```diff
@@ -27,9 +27,8 @@
 def validate_json(f):
     @wraps(f)
     def wrapper(*args, **kw):
-        try:
-            request.get_json(force=True)
-        except:
+        json_data = request.get_json(force=True)
+        if isinstance(json_data, basestring):
```

Comment:
there are some cases when request.get_json return None

Reply:
Could you give an example?

Reply:
When calling request.get_json() with force=False and request mime type is not json

Reply:
We are calling with `force=True`, so this case does not apply to us.